pkg/cli-runtime/printer/tabwriter/tabwriter.go

```diff
@@ -319,6 +322,10 @@ func (b *Writer) writeLines(pos0 int, line0, line1 int) (pos int) {
 		// if TabIndent is set, use tabs to pad leading empty cells
 		useTabs := b.flags&TabIndent != 0
 
+		if len(line) > 0 && b.paddingStart > 0 {
```

Comment:
why do you need to check whether paddingStart is above 0 ? If its 0 then there is no padding anyway